README.md

```diff
@@ -1,6 +1,8 @@
 GitLink
 ==========
 
+[![Build status](https://ci.appveyor.com/api/projects/status/g56b4xyl4q0ean8i/branch/develop?svg=true)](https://ci.appveyor.com/project/AArnott/gitlink/branch/develop)
```

Comment:
@gep13: appveyor may try, even without a yml file. But it will likely fail if the repo isn't super simple (which IIRC it isn't).